deployments/ansible/roles/collector/defaults/main.yml

```diff
@@ -1,16 +1,16 @@
 ---
 # defaults file for splunk-otel-collector role vars
-
+# Note: To modify vars based upon ansible_os_family  change the values of vars in tasks/vars.yml
 # splunk_access_token is REQUIRED variable
 splunk_access_token: ""
 
 splunk_realm: us0
 
 splunk_otel_collector_version: latest
 
-# Set this variable to /etc/otel/collector/gateway_config.yaml in order to
+# Set this variable to PATH/gateway_config.yaml in order to
 # deploy splunk-otel-collector in gateway mode
-splunk_otel_collector_config: /etc/otel/collector/agent_config.yaml
+splunk_otel_collector_config: ""
```

Comment:
Lets add comments here with default values for both platforms

Reply:
Okay.